common.sh

```diff
@@ -210,7 +210,8 @@ export KNI_INSTALL_FROM_GIT=${KNI_INSTALL_FROM_GIT:-}
 #
 # if we provide OPENSHIFT_RELEASE_IMAGE, do not curl. This is needed for offline installs
 if [ -z "${OPENSHIFT_RELEASE_IMAGE:-}" ]; then
-  LATEST_CI_IMAGE=$(curl https://openshift-release.svc.ci.openshift.org/api/v1/releasestream/4.6.0-0.ci/latest | grep -o 'registry.svc.ci.openshift.org[^"]\+')
+  OPENSHIFT_RELEASE_STREAM=${OPENSHIFT_RELEASE_STREAM:-4.6}
```

Comment:
I would have expected this to include the z portion of the version, too. Are the 4.5.1 CI update releases part of the 4.5.0 release stream?

Reply:
It appears from https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/ that there is only one CI for each y-stream. So e.g. CI on the release-4.4 branch ends up in the 4.4.0-0.ci stream, even though we've already passed 4.4.9 in terms of releases.